config/environments/test.rb

```diff
@@ -49,4 +49,8 @@
 
   # Use separate cache stores for parallel_tests
   config.cache_store = :file_store, Rails.root.join("tmp", "cache", "paralleltests#{ENV['TEST_ENV_NUMBER']}")
+
+  # Disables automatic creation of the test DB so that it has to be explicitly
+  # generated before running tests.
+  config.active_record.maintain_test_schema = false
```

Comment:
no it isn't. I always need to do that when there are new migrations, develop or rails4... foreman or any other rails application... what am I missing?